Translation corrections (#166)
07th-mod/higurashi-console-arcs: Update/tara_001.txt

//rまったく、しょうがないやつだよな――それに苦笑で答えようとした途端。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まったく、しょうがないやつだよな——それに苦笑で答えようとした途端。",

> 			NULL, "And just as I made I wry smile and was about to comment on how helpless Shion was.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And just as I made a wry smile and was about to comment on how troublesome Shion was.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s17/03/110300028", 256, TRUE);
	OutputLine(NULL, "「なーんて。",

> 			NULL, "\"Just kidding that?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Just kidding.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 3, "ps3/s17/03/110300029", 256, TRUE);
	OutputLine(NULL, "おじさん、気のまわしすぎかなぁ～？",

> 			NULL, " Maybe this old man is getting to skeptical for his own good~?", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " Maybe this old man is getting too skeptical for his own good~?", Line_WaitForInput);
```

//r魅音がポケットからマジックを引きぬく。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "魅音がポケットからマジックを引きぬく。",
			NULL, "Mion pulled a felt-tip marker out of her pocket.", Line_Normal);
	ClearMessage();

//r…あぁ、顔にイタズラのアレだな。油性は辛いよ。せめて、水性にしてあげないか？
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…あぁ、顔にイタズラのアレだな。油性は辛いよ。せめて、水性にしてあげないか？",

> 			NULL, "...Aah, and she has a mischievous look on her face. Permanent marker is too harsh. Couldn't she show him mercy and use a washable one?", Line_Normal);

Reviewer: ```suggestion
			NULL, "...Aah, and she has a mischievous look on her face. Permanent marker is too harsh. Couldn't she show him some mercy and use a washable one?", Line_Normal);
```
